### DISCUSSION TITLE
Could you lower the price  please?

### DISCUSSION THREAD

**Discussion by martianmartian** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#860] [General]
Really like your app. 

it's just... the only feature i use is charging limit. would like to contribute financially but not really into other features for 20 bucks...

how about 5 bucks for just charging limit, and discharge, with no ads?

**Comment by MatthiasKerbl** [comment]
Hi @martianmartian,

I am glad you like our app AlDente.
If you want to support the development of AlDente but don't want to buy the lifetime license, you can also buy the subscription license for less than half the price.
AlDente Free is completely free and has no advertisements and offers exactly the features you mentioned, Charge Limiter and Discharge.

Cheers,
Matthias

**Comment by martianmartian** [comment]
I understand and appreciate your hard work making this app. Just a curious discussion. hope you don't mind. 

Seems to me that keeping track of all the monthly fees burdens me, feels like leeches on my wallet ... i'm also working on an app, so i avoided this type of charging mechanism. 

Seems like more reasonable to have this feature install on mac for once and not have to think about it again. what makes you think people would want to pay you monthly?

**Comment by asoksevil** [comment]
Is there any promotion going on anytime soon? I like the app however I also struggle to justify 20 for basically for the charging limit (life time license, not a fan of subscription either)

Many thanks

**Comment by romancin** [comment]
Are you applying a discount this year for black friday like you did last year? Thank you!

**Comment by MatthiasKerbl** [comment]
Hi everyone,

Since this issue turned into a discussion about the price of software in general, I will convert it into a discussion.

Cheers,
Matthias
